src/main/java/eu/xenit/alfred/initializr/app/StartApplication.java

```diff
@@ -40,12 +42,17 @@ public HomeController homeController(InitializrMetadataProvider metadataProvider
     }
 
     @Bean
-    public CustomProjectGenerationInvoker projectGenerationInvoker(
-            ApplicationContext applicationContext,
-            ApplicationEventPublisher eventPublisher,
-            ProjectRequestToDescriptionConverter projectRequestToDescriptionConverter) {
-        return new CustomProjectGenerationInvoker(applicationContext, eventPublisher,
-                projectRequestToDescriptionConverter);
+    public CustomProjectGenerationInvoker projectGenerationInvoker(ApplicationContext applicationContext) {
+        return new CustomProjectGenerationInvoker(applicationContext,
+                new DefaultProjectRequestToDescriptionConverter());
     }
 
+    @Bean
+    @ConditionalOnMissingBean
+    ProjectGenerationController projectGenerationController(InitializrMetadataProvider metadataProvider,
+            ApplicationEventPublisher eventPublisher,
```

Comment:
eventPublisher and applicationContext no longer needed as params here

Reply:
removed